aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -256,5 +401,48 @@ Tensor& bernoulli_scalar_cuda_(Tensor &self, double p, Generator* gen) {
   return self;
 }
 
+static void uniform_kernel_cuda(TensorIterator& iter, double from_, double to_, Generator* gen) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "uniform_cuda", [&] {
+    auto from = static_cast<scalar_t>(from_);
+    auto to = static_cast<scalar_t>(to_);
+    AT_CHECK(from <= to,
+      "uniform_ expects to return a [from, to) range, but found from=", from,
+      " > to=", to);
+    AT_CHECK((to - from) <= std::numeric_limits<scalar_t>::max(),
+          "uniform_ expects to-from ≤ std::numeric_limits<double>::max(), but found to=", to,
+          " and from=", from, " which result in to-from to exceed the limit");
+
+    using accscalar_t = at::acc_type<scalar_t, true>;
```

Comment:
ugh, I should have never let us define `acc_type` with a boolean template parameter lol